crates/sui/offline_signing.md

```diff
@@ -0,0 +1,62 @@
+# Offline Signing
+
+This is a guide for users who wish to sign a transaction using an offline device without relying on sui keystore. Here are the steps to:
+ 1. Serialize the data for signing;
+ 2. Take the serialized data elsewhere to sign (wallet of your choice, or tools in other languages) to produce a signature and the corresponding public key.
+ 3. Execute the signed transaction.
+
+## Step 1: Serialize a Transfer
+
+A transaction data must be serialized according to [BCS](https://crates.io/crates/bcs). It is supported in [other languages](https://github.com/zefchain/serde-reflection#language-interoperability).
+
+Here an example is provided to serialize a transfer data in CLI. This outputs a serialized transaction data in Base64.
+
+```shell
+sui client serialize-transfer-sui --to 0x581a119a6576d3b502b5dc47c5de497b774e68ca --sui-coin-object-id 0x0599b794da39169f7c75d34eba06ae105fedc61b --gas-budget 1000
+VHJhbnNhY3Rpb25EYXRhOjoAA1gaEZpldtO1ArXcR8XeSXt3TmjKAFgaEZpldtO1ArXcR8XeSXt3TmjKBZm3lNo5Fp98ddNOugauEF/txhsCAAAAAAAAACC0knjIoZEdbQBQuqg3feG/GA0L2v9gLDfH2uX8iGf5SwEAAAAAAAAA6AMAAAAAAAA=
+```
+
+## Step 2: Sign the data
+This can be done elsewhere in your signing device or implemented in languages of your choice. Sui accepts signature for both Secp256k1 and Ed25519 correponding to the specified flag field. 
+
+An accepted Secp256k1 signature follows:
+1. The signature must be of length 65 bytes in the form of `[r, s, v]` where the first 32 bytes is `r`, the seconds 32 bytes is `s` and the last byte is `v`. 
+2. The `r` value can be between 0x1 and 0xFFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFE BAAEDCE6 AF48A03B BFD25E8C D0364140 (inclusive). 
+3. The `s` value must be in the lower half of the curve order, i.e. between 0x1 and 0x7FFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF 5D576E73 57A4501D DFE92F46 681B20A0 (inclusive). 
+4. The `v` represents the recovery ID, which must be normalized to 0, 1, 2 or 3. Note that unlike [EIP-155](https://eips.ethereum.org/EIPS/eip-155) chain ID is not used to calculate the `v` value. 
```

Comment:
not sure if referring to EIP 155 is appropriate, I cannot find RFC or standard on v hence provided here as an example

Reply:
It's fine.